ClearKeep/Scenes/Chat/Views/SelectFileView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SelectFileView.swift
+//  ClearKeep
+//
+//  Created by đông on 16/04/2022.
+//
+
+import SwiftUI
+
+struct SelectFileView: View {
+    var body: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/23bd2623ae2e69d9588056da830b6b5158000cf8/ClearKeep/Scenes/Chat/Views/SelectFileView.swift#L11"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>SelectFileView.swift:11</code></td>
    </tr>
  </tbody>
</table>